cpp/benchmarks/CMakeLists.txt

```diff
@@ -330,7 +330,7 @@ ConfigureNVBench(CSV_WRITER_NVBENCH io/csv/csv_writer.cpp)
 
 # ##################################################################################################
 # * ast benchmark ---------------------------------------------------------------------------------
-ConfigureBench(AST_BENCH ast/transform.cpp)
+ConfigureNVBench(AST_BENCH ast/transform.cpp)
```

Comment:
IIRC, nightly benchmarks rely on the proper suffix to distinguish between nvbench and gbench so we may want to update this to `AST_NVBENCH` as well.

Reply:
alright